### PR TITLE
Chapter 4: Rails-flavored Ruby

### PR DESCRIPTION
**Pull request by tungmk97**

### CODE REVIEW

**Comment by tungmk97** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title(page_title = '')
```
Comment (DuongNT1512) [inline app/helpers/application_helper.rb]: remove ()
double quote

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= t ".title" %></title>
+    <title><%= full_title(yield(:title)) %></title>
```
Comment (DuongNT1512) [inline app/views/layouts/application.html.erb]: remove () thừa

File: app/views/static_pages/home.html.erb [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<% provide:title, t("static_pages.home") %>
+<% provide:title, t("static_pages.home.home") %>
```
Comment (DuongNT1512) [inline app/views/static_pages/home.html.erb]:
```suggestion
<% provide :title, t("static_pages.home.home") %>
```

**Comment by DuongNT1512** [comment]
commented

**Comment by tungmk97** [comment]
ready

File: app/helpers/application_helper.rb [hunk 4]
```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ''
```
Comment (DuongNT1512) [inline app/helpers/application_helper.rb]:
```suggestion
  def full_title page_title = ""
```

**Comment by DuongNT1512** [comment]
commented

**Comment by tungmk97** [comment]
ready

**Comment by DuongNT1512** [comment]
merged

**Pull request merged**